iphone/Maps/Classes/ClipboardMonitor.mm

```diff
@@ -0,0 +1,116 @@
+#import "MapsAppDelegate.h"
+#import "ClipboardMonitor.h"
+#import "UIKitCategories.h"
+#import "MapsAppDelegate.h"
+#import <CoreApi/DeepLinkParser.h>
+#include <CoreApi/Framework.h>
+#import <CoreApi/DeepLinkData.h>
+#import "UIKitCategories.h"
+#import <CoreApi/DeepLinkSearchData.h>
+
+@interface ClipboardChecker ()
+
+@property (nonatomic, strong) NSTimer *timer;
+@property (nonatomic, copy) NSString *previousClipboardText;
+@property (nonatomic, assign) BOOL isErrorPromptShown;
+
+@end
+
+@implementation ClipboardChecker
+
+NSString *privacyProxy = @"https://url-un.kartikay-2101ce32.workers.dev/coordinates?url=%@";
+NSString *localizedStringOk = NSLocalizedString(@"ok", @"");
+- (void)startChecking:(UIWindow *)window {
+    UIPasteboard *pasteboard = [UIPasteboard generalPasteboard];
+    NSString *currentClipboardText = pasteboard.string;
+
+    [pasteboard setString:@""];
```

Comment:
What if someone loses something important in the clipboard when opening OM?

Reply:
I have changed that in the updated code, the clipboard is only cleared if it is a google maps url.